shinydashboard/lantern/server.R

```diff
@@ -121,26 +126,58 @@ function(input, output, session) { #nolint
 
   output$show_filters <- renderUI({
     if (show_filter()) {
-      fluidRow(
-        column(width = 4,
-          selectInput(
-            inputId = "fhir_version",
-            label = "FHIR Version:",
-            choices = isolate(app$fhir_version_list()),
-            selected = ui_special_values$ALL_FHIR_VERSIONS,
-            size = 1,
-            selectize = FALSE)
-        ),
-        column(width = 4,
-          selectInput(
-            inputId = "vendor",
-            label = "Developer:",
-            choices = app$vendor_list,
-            selected = ui_special_values$ALL_DEVELOPERS,
-            size = 1,
-            selectize = FALSE)
+      if (show_availability_filter()) {
+        fluidRow(
+          column(width = 4,
+            selectInput(
```

Comment:
Is it possible to pull out these selectInputs and set them to variables in this function? I'm not sure if that's allowed in R but it would be nice to not have to repeat the whole vendor & fhir version dropdowns twice